recipes/ace/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "6.4.3" %}
+
+{% set dyn_ext = ".so" %}  # [linux]
+{% set dyn_ext = ".dylib" %}  # [osx]
+{% set dyn_ext = ".dll" %}  # [win]
+
+package:
+    name: ace
+    version: {{ version }}
+
+source:
+    fn: ace-{{ version }}.tar.bz2
+    url: http://download.dre.vanderbilt.edu/previous_versions/ACE-{{ version }}.tar.bz2
+    sha256: b80e7b8137d48dc68b281233417fe19b21a904ea642a05ea795bd6a546bcbb1d
+    patches:
+      #! /usr/bin/perl -> #! /usr/bin/env perl
+      - linux_install.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - perl
```

Comment:
Should pin this exactly to one `perl` version. Otherwise `conda-build-all` and `conda-smithy` will create a matrix of `perl` builds.